src/chapters/enums/enums-in-models.rst

```diff
@@ -14,3 +14,241 @@ The final code from this video is in the `enums branch <https://github.com/Launc
+
+With the template variable ``types`` now defined, we can use our ``EventType`` values in the view.
+
+Use Enum Value in a ``Select`` Element
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The list of constants returned from ``EventType`` lends itself well to a ``select``-type form 
+input. We'll update our form so that a user will have the option to choose one of the provided 
+event types from a dropdown menu.
+
+In ``templates/events/create.html``:
+
+.. sourcecode:: html
+   :lineno-start: 27
+
+   <div class="form-group">
+      <label> Type
+      <select th:field="${event.type}">
+         <option th:each="type : ${types}"
+                  th:value="${type}"
+                  th:text="${type.displayName}"
+         ></option>
+      </select>
+      </label>
+   </div>
+
+As with the other form inputs on the page, the ``th:field`` attribute determines the ``name``
+and ``id`` attributes for the ``select`` tag. We make an ``option`` tag for each of the ``EventType``
+values, making use of the ``types`` variable we passed in from the controller in 
+:ref:`the previous step <passing-enums-through-controller>`. We set the ``value`` attribute for the 
+model data to be the ``EventType`` value using ``th:value``. And the type name shown to the user 
+of the form as the ``displayName`` of the type, using ``th:text``.
+
+Use Enum Properties to Display Information
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once an event is created, to display its ``type`` field in the table of all events, we'll modify 
+``templates/events/index.html`` to include another column:
+
+.. sourcecode:: html
+   :lineno-start: 15
+
+   <!-- other table headers -->
+   <th>Type</th>
+   <!-- other event data -->
+   <td th:text="${event.type.displayName}"</td>
+
+In this case, the type displayed is the value of the event object's ``type`` field, so the controller 
+method responsible for rendering this view does not need a ``types`` variable passed in. To show the 
+more user-friendly view of the type value, we use the ``.displayName`` field of ``EventType``.
+
+Check Your Understanding
+------------------------
+
+.. admonition:: Question
+
+   When we add a field to the ``EventType`` enum from ``coding-events``, what is the strongest 
+   reason why we don't we write a setter method for that field?
+
+   #. Enum classes cannot contain setter methods
+   #. Final variables cannot be reassigned
+   #. Enum fields cannot be reassigned
+   #. We don't use a setter method in the rest of the application
+
+.. ans: b, Final variables cannot be reassigned
+
+.. admonition:: Question
+
+   In ``coding-events``, say we change our template variable name in 
+   ``EventController.displayCreateEventForm`` so that ``EventType.values()`` is now assigned to 
+   a variable, ``categories``. Which of the template expressions in the following codeblock 
+   from ``create.html``, if any, will be changed? Select all that apply.
```

Comment:
"will be changed?" -> "should be changed to reflect this update?" (or similar)

Reply:
More context, aside from clarifying the question a bit more, C, D, and part of B don't technically *need* to be changed, but should be. In other words, we could continue to define the loop variable as `type`.

Reply:
yeah, i thought about this ambiguity when writing it but depending on how they interpret option b) will determine how they respond to the next 2 options. i changed the answers to also read "should be changed.." but maybe it still needs clarifying